types/oracledb/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "dotenv": "^8.2.0"
+    }
+}
```

Comment:
I think this package doesn't need `dotenv`.

Reply:
I'm not familiar with the package or its tests, but [dotenv is being used](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6ec670c7d0220af69ba7dd67b386e3d23e00874b/types/oracledb/oracledb-tests.ts#L2)

Reply:
Whoops, you are right. My bad!